src/main/java/com/example/demo/database/Receipt.java

```diff
@@ -15,13 +15,14 @@ public class Receipt {
 
     @Id
     @GeneratedValue(strategy = GenerationType.AUTO)
-    private long receiptId;
+    private long id;
 
     private long customerId;
 
     private int totalAmount;
 
-    private Date dateTime;
+    @Column
+    private LocalDate date;
```

Comment:
What date is it? Is it the date of creation, or the date of update?
You should be more specific in you naming